### ISSUE TITLE
CMake install target doesn't provide imconfig.h

### ISSUE THREAD

**Issue by barsoosayque** [eliasdaler/imgui-sfml#75]
On `develop` branch, when you `$ cmake --build . --target install` the library, there is just no `imconfig.h` provided (not an empty one). So, without a clean Imgui in the prefix, I cannot use the library in my code. I can see two solutions:

1. Bundle `imconfig-SFML.h` as a `imconfig.h`, since original config file is empty;
2. Bundle original `imconfig.h`, and a user should include `imconfig-SFML.h` themself.

Now, I think that the first solution is ultimatly the better one, because installation of this library is provided with Imgui headers, so I don't see any reason not to do that. It's just convenient, after all.

**Comment by eliasdaler** [comment]
Agreed on 2. Working on it, plus I'll add an option to specify if you want to use imconfig-SFML.h or your custom config file.
